packages/pg-gateway/src/connection.types.ts

```diff
@@ -13,9 +13,18 @@ export type TlsInfo = {
   sniServerName?: string;
 };
 
+export const ServerStep = {
+  AwaitingInitialMessage: 'AwaitingInitialMessage',
+  PerformingAuthentication: 'PerformingAuthentication',
+  ReadyForQuery: 'ReadyForQuery',
+} as const;
```

Comment:
I realize this was just moved, but wondering if it you intentionally don't use enums for this?

Reply:
I avoid any TypeScript feature which alters the runtime code. IMHO TypeScript should only erase types. (especially with new proposals like https://nodejs.org/fr/blog/release/v22.6.0#experimental-typescript-support-via-strip-types)